### ISSUE TITLE
Error: Process exited with non-zero code: 1

### ISSUE THREAD

**Issue by codiceovvio** [AtomLinter/linter-shellcheck#73]
Hello,
I am on Ubuntu 16.04 - 64-bit (everything up-to-date daily), with Atom 1.10.2, and I am getting the following error:
```
Error: Process exited with non-zero code: 1
    at ChildProcess.<anonymous> (/home/mare/.atom/packages/linter-shellcheck/node_modules/sb-exec/lib/index.js:60:22)
    at emitTwo (events.js:100:13)
    at ChildProcess.emit (events.js:185:7)
    at maybeClose (internal/child_process.js:850:16)
    at Socket.<anonymous> (internal/child_process.js:323:11)
    at emitOne (events.js:90:13)
    at Socket.emit (events.js:182:7)
    at Pipe._onclose (net.js:477:12)
```
It started today after updating apm packages, which updated also linter-shellcheck.
Do not know if you may need more info, I am getting this for every .sh file I open or edit.

Thaks a lot to all of you for giving your time to develop all linters, and sorry for my english.
Marcello

**Comment by sdemura** [comment]
I'm getting the same thing with brew installed shellcheck on OS X

**Comment by Arcanemagus** [comment]
I really wish all repos had proper specs... pushing out a patch for this in a few minutes.

**Comment by Arcanemagus** [comment]
v1.3.0 should fix this for you, it also should show better error ranges as it will only highlight the next "word" (as `language-puppet` defines it), instead of the entire line.

**Comment by codiceovvio** [comment]
Updated to v1.4.0 now, and errors are gone!
Thanks a lot again!
